sparse/utils.py

```diff
@@ -2,7 +2,7 @@
 from .core import COO
 
 
-def assert_eq(x, y):
+def assert_eq(x, y, rtol=1.e-5, atol=1.e-8):
```

Comment:
Should we just pass through `**kwargs` here?

Reply:
Yes, that seems like a better option.